### ISSUE TITLE
Asian characters in the title cause problems with export

### ISSUE THREAD

**Issue by Anongh** [zadam/trilium#746]
If Chinese, Japanese, and Korean characters are in the title, the tar compression file is tangled with "PaxHeader" and does not get the desired result.

Test yourself with the following characters:
簡體字
かんじ
한글

I'm using Trilium on Windows 8.1.

```
Versions:
App version:0.37.6
DB version:153
Sync version:11
Build date:2019-11-26T22:50:08+01:00
Build reception:5193f073e9e55f5440fe2e71fbd2cdb2d2c6b

Part of the log file:
09:47:13.744 INFO GET/api/notes/LCVKgx3B0F6n/export/subtree/html/1.0/5MuAfCk2Lz?1575420433727
09:47:13.812 INFO Sending Message to all clients: {"type":"task-succed", "taskId":"5MuAfCk2Lz", "taskType":"export"
```

**Comment by zadam** [comment]
When you see "PaxHeader" it means you're using tool without support for the "pax" tar extension.

For me specifically Gnome's Archive Manager can display tar file with UTF-8 filenames correctly:

![image](https://user-images.githubusercontent.com/617641/70182151-90be4f80-16e3-11ea-83e1-c4db47f715e1.png)

**Comment by Anongh** [comment]
Thanks for the answer.

I tried to install Gnome's Archive Manager as you advised, but it was hard on Windows, so I looked for alternatives on behalf of my 7zip.
https://alternativeto.net/software/7-zip/

Among the alternatives, WinRAR was the only one that accurately showed PaxHeader, and the rest of the compression programs did not express PaxHeader.

I was curious about why this problem occurred in Windows and I did a search.
Japanese analysis of tar decompression in Windows was the only value to refer to.
https://www.webimpact.co.jp/banchoblog/?p=709

I close this issue.
Thank you
